src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.html

```diff
@@ -63,50 +63,74 @@ <h1><b>Ticket Number: {{ lastUpdatedJJDispute.ticketNumber }} </b></h1>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Location of Offence</p>
-            <p class="section-grid-text"> {{ lastUpdatedJJDispute.offenceLocation }}</p>
-          </span>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Organization/Detachment/Location</p>
-            <p class="section-grid-text">{{ lastUpdatedJJDispute.policeDetachment }}</p>
-          </span>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Violation Date</p>
-            <p class="section-grid-text">{{ lastUpdatedJJDispute.issuedTs | date: "MMM dd, yyyy HH:mm" : "UTC" }}</p>
-          </span>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Violation Time</p>
-            <p class="section-grid-text">{{ lastUpdatedJJDispute.issuedTs | date: "HH:mm" : "UTC" }}</p>
-          </span>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Online Submission Date</p>
-            <p class="section-grid-text">{{ lastUpdatedJJDispute.submittedTs | date: "MMM dd, yyyy" : "UTC" }}</p>
-          </span>
-          <span class="section-grid-cell">
-            <p class="section-grid-header">Date Received from ICBC</p>
-            <p class="section-grid-text">{{ lastUpdatedJJDispute.icbcReceivedDate | date: "MMM dd, yyyy" }}</p>
-          </span>
-          <ng-container *ngIf="type !== 'ticket'">
+        <form [formGroup]="ticketInformationForm">
+          <div style="display: grid; grid-template-columns: auto auto auto auto">
             <span class="section-grid-cell">
-              <p class="section-grid-header">Court Location Code</p>
-              <p class="section-grid-text">{{ lastUpdatedJJDispute.courtAgenId }}</p>
+              <p class="section-grid-header">Surname</p>
+              <p class="section-grid-text" *ngIf="!isEditMode">{{ lastUpdatedJJDispute.occamDisputantSurnameNm }}</p>
+              <mat-form-field appearance="outline" *ngIf="isEditMode">
+                <input matInput formControlName="occamDisputantSurnameNm">
+                <mat-error *ngIf="ticketInformationForm.controls.occamDisputantSurnameNm.hasError('maxlength')">{{ "error.max_length" | translate }}30</mat-error>
+              </mat-form-field>
             </span>
             <span class="section-grid-cell">
-              <p class="section-grid-header">Court Location</p>
-              <p class="section-grid-text">{{ lastUpdatedJJDispute.courthouseLocation }}</p>
+              <p class="section-grid-header" *ngIf="!isEditMode">Given Name(s)</p>
+              <p class="section-grid-text" *ngIf="!isEditMode">{{ lastUpdatedJJDispute.occamDisputantGivenNames }}</p>
+              <p class="section-grid-header" *ngIf="isEditMode">Given Name 1</p>
+              <mat-form-field appearance="outline" *ngIf="isEditMode">
+                <input matInput formControlName="occamDisputantGiven1Nm">
```

Comment:
originally was splitting it in frontend, are we going to change it ?

Reply:
The UI displays all three fields in one long string. That is not changing. But for editing the fields, we should be editing the individual fields that make up the one long string (occamDisputantGiven1Nm + occamDisputantGiven2Nm + occamDisputantGiven3Nm). This will provide the admin staff with the best tools to customize the name properly. Otherwise, we'd have to manually split the long text into space separated words and put one word into each of the 3 names fields. But then we run into issues if there happens to be a name with a space, like Anna Marie as one name. Best to let staff set it exactly as they need to.

Reply:
Cool